### DISCUSSION TITLE
asus firmware

### DISCUSSION THREAD

**Discussion by xvtstephen** [samveen/mt7902-dkms#21] [General]
hello, 
I am new at this so please, excuse my ignorance. I downloaded firmware from asus for my asus vivobook go 15.
the build and install does not throw an error. do I need to change anything (names) as I did not use Acer firmware.
or the linux code only looks for /lib/firmware/mediatek/

**Comment by samveen** [comment]
Please extract the firmware and examine it for the firmware blob, and place it at the expected path with the expected name, and try.

Don't forget to list the results of you experiment here please. It may help all the rest spot issues.

**Reply by xvtstephen** [reply]
there are many folders under mediatek called mt9999 plus files in that folder. I will create mt7902 and put all files there.
please, clarify what is the blob and what do I do with it. appreciate your efforts.

**Comment by xvtstephen** [comment]
I copied the asus firmware into mt7902 and it does not load the module. still says no network devices.
is there anything else I can try?
